tests/python/openassetio/hostApi/test_managerfactory.py

```diff
@@ -134,3 +135,53 @@ def assert_expected_host(hostSession):
         # confirm
 
         assert actual == expected
+
+
+class Test_ManagerFactory_createManager:
+    def test_implementation_factory_used(
+            self, a_manager_factory, mock_manager_interface_factory, mock_manager_interface):
+        # setup
+
+        expected_identifier = "a.manager"
+        mock_manager_interface_factory.mock.instantiate.return_value = mock_manager_interface
+
+        # action
+
+        manager = a_manager_factory.createManager(expected_identifier)
+
+        # confirm
+
+        # TODO(DF): This can be changed to openassetio.hostApi.Manager
+        #  once C++ migration of Manager is complete.
+        assert isinstance(manager, _openassetio.hostApi.Manager)
```

Comment:
[consider] Moving this to its own test in the spirit of "test cases describe expected behaviour"

Reply:
Split instance type check and config check in 227da8cb